packages/core/src/mcp/constants.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2025 Qwen
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * OAuth client name used for MCP dynamic client registration.
+ * This name must match the allowlist on MCP servers like Figma.
+ */
+export const MCP_OAUTH_CLIENT_NAME = 'Gemini CLI MCP Client';
```

Comment:
Good design. Once more MCP authentications support Qwen Code, we will update this MCP_OAUTH_CLIENT_NAME.